kontrakter/vegtrafikkulykke/trafikkulykke/1.0/trafikkuhell.schema.json

```diff
@@ -0,0 +1,637 @@
+      "enum": [
+        "TIL_FRA_ARBEID",
+        "I_ARBEID",
+        "TIL_FRA_SKOLE",
+        "TIL_FRA_BUTIKK_BANK_ETC",
+        "FRITIDSREISE",
+        "OEVELSESKJOERING",
+        "LEK",
+        "HJEM_PAA_PERMISJON",
+        "TIL_FRA_FEST",
+        "ANNET_FORMAAL",
+        "UKJENT"
+      ]
+    },
+    "person": {
+      "type": "object",
+      "description": "Informasjon om person involvert i ulykken",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gjør personen unikt identifiserbar innad i saken, må være lik ved oppdateringsmelding."
+        },
+        "kjoenn": {
+          "type": "string",
+          "enum": ["MANN", "KVINNE"]
+        },
+        "foedselsdato": {
+          "type": "string",
+          "format": "date"
+        },
+        "plasseringKjoeretoey": {
+          "$ref": "#/definitions/plasseringKjoeretoey"
+        },
+        "skadegrad": {
+          "$ref": "#/definitions/skadegrad"
+        },
+        "beskyttelsesutstyr": {
+          "$ref": "#/definitions/beskyttelsesutstyr"
+        },
+        "skadeforloep": {
+          "$ref": "#/definitions/skadeforloep"
+        },
+        "ferdselsformaal": {
+          "$ref": "#/definitions/ferdselsformaal"
+        },
+        "beskrivelsePersonskade": {
+          "type": "string"
+        },
+        "postnummer": {
+          "type": "integer"
+        },
+        "poststed": {
+          "type": "string"
+        }
+      },
+      "required": ["id"],
+      "additionalProperties": false
+    },
+    "uhellPlassering": {
+      "description": "Informasjon om hvor ulykken skjedde",
+      "type": "object",
+      "properties": {
+        "veg": { "$ref": "#/definitions/vegInformasjon" },
+        "kryssendeVeg": { "$ref": "#/definitions/vegInformasjon" },
+        "beskrivelse": {
+          "type": "string"
+        },
+        "koordinater": { "$ref": "#/definitions/koordinater" }
+      },
+      "required": ["veg", "kryssendeVeg", "beskrivelse"],
+      "additionalProperties": false
+    },
+    "vegInformasjon": {
+      "description": "Informasjon om vei",
+      "type": "object",
+      "properties": {
+        "gate": { "type": "string" },
+        "kommune": { "type": "string" },
+        "vegkategori": { "$ref": "#/definitions/vegkategori" },
+        "vegnummer": { "type": "number" }
+      },
+      "required": ["gate", "kommune", "vegkategori", "vegnummer"],
+      "additionalProperties": false
+    },
+    "vegkategori": {
+      "type": "string",
+      "enum": [
+        "EUROPAVEG",
+        "RIKSVEG",
+        "FYLKESVEG",
+        "KOMMUNAL_VEG",
+        "PRIVAT_VEG",
+        "ANNET"
+      ]
+    },
+    "koordinater": {
+      "description": "Koordinater som angir et visst sted",
+      "type": "object",
+      "properties": {
+        "koordinatOest": { "type": "string" },
+        "koordinatNord": { "type": "string" },
+        "koordinatsystem": { "type": "string" }
```

Comment:
Her bør vi bli definere hvordan koordinatsystem skal oppgis, eks. med SRID. Vi bruker for eksempel SRID [25833](https://epsg.io/25833)  aka ETRS89 / UTM zone 33N.
Da kan egenskapen kanskje hete `"srid": { "type": "integer" }` 🤔